packages/ember/translations/en-us.json

```diff
@@ -52,6 +52,9 @@
         "match": "This field is invalid",
         "doesNotMatch": "This field is invalid"
       }
+    },
+    "select": {
+      "baseDefaultText": "Select an Option"
```

Comment:
```suggestion
      "defaultText": "Select an Option"
```

Reply:
Also, can this block be moved above `validation`? To keep everything alphabetical